gui/src/app/Scripting/Runtime/makePyRuntime.ts

```diff
@@ -0,0 +1,65 @@
+import { ProjectDataModel } from "@SpCore/ProjectDataModel";
+
+import spPreamble from "./preamble.py?raw";
+import spRunData from "./run_data.py?raw";
+import spLoadConfig from "./load_args.py?raw";
+import spCmdStan from "./cmdstan.py?raw";
+import spRunSampling from "./sample.py?raw";
+import spDrawsScript from "../pyodide/sp_load_draws.py?raw";
+import spRunAnalysis from "./run_analysis.py?raw";
+
+const indent = (s: string) => {
+  return s
+    .trim()
+    .split("\n")
```

Comment:
It wouldn't be valid for input data to include data fields with embedded newlines, right?

Reply:
We're only calling this on text we completely control at the moment, but this is worth worrying about if the usage of this function changes